sources/packages/api/src/utilities/datatable-config.ts

```diff
@@ -26,3 +26,12 @@ export enum SortPriority {
   Priority3 = 3,
   Priority4 = 4,
 }
+
+/**Pagination Query param constants **/
+export enum PaginationParams {
+  SortField = "sortField",
+  SortOrder = "sortOrder",
+  Page = "page",
+  PageLimit = "pageLimit",
+  SearchCriteria = "searchCriteria",
+}
```

Comment:
All constants in an enum should belong to the same family. Example an enum of colors will be Green, Red, Blue. Is this really a valid enum ?

Reply:
Yes @henryoforeh-aot . This enum is to provide consistency to query param names used for pagination, sort and search of a data set across all APIs.